### DISCUSSION TITLE
Custom perturbations for I/Q samples

### DISCUSSION THREAD

**Discussion by UtkuGit** [NVlabs/sionna#372] [Q&A]
I would like to change the phase and amplitudes of I/Q samples with user defined values. I can see in the Hello World example that an AWGN channel adds perturbations with a specific Eb/N0, but I wonder if there is a way to manually alter the phase and amplitude of I/Q samples. I would appreciate your help.

**Comment by jhoydis** [comment]
Hi @UtkuGit,

Yes, you can very easily add such perturbations. If you have a tensor `y` containing IQ samples, you can simply multiply it by a tensor `z` that represents amplitude and phase terms:
```
y_altered = y*z
```
That's all.
